regression/contracts/assigns_enforce_statics_subfunctions/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--enforce-contract bar _ --pointer-primitive-check
+^EXIT=10$
+^SIGNAL=0$
+^\[foo.\d+\] line \d+ Check that y is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that foo\$\$1\$\$x is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that \*y is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that \*yy is assignable: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+Checks whether static local and global variables are correctly tracked
+in assigns clause verification, accross subfunction calls.
```

Comment:
nit:
```Makefile
in assigns clause verification, across sub-function calls.
```

Reply:
I think it's safe to use subfunction
https://www.merriam-webster.com/dictionary/subroutine

Reply:
It's fine to use subroutine, the core issue was the typo: `s/accross/across/g`.